Backend/src/test/java/com/senla/repository/EventRepositoryTest.java

```diff
@@ -0,0 +1,27 @@
+package com.senla.repository;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.senla.model.Event;
+import com.senla.model.User;
+import java.util.Optional;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+
+class EventRepositoryTest extends DatabaseTest {
```

Comment:
minor: usually test created for the service layer because there will be the main logic or for the contoller or for both service and controller. 

For the repository layer integration tests need if you have some complex query with joins for example.
But for the most part it's just bacis default CRUD operations that doesn't make much sense to test.

It's fine for the next part I suggest to create integration tests for the service layer.